apps/src/gamelab/GameLab.js

```diff
@@ -153,13 +152,31 @@ GameLab.prototype.init = function (config) {
   config.dropletConfig = dropletConfig;
   config.appMsg = msg;
 
+  var onMount = function () {
+    config.loadAudio = this.loadAudio_.bind(this);
+    config.afterInject = this.afterInject_.bind(this, config);
+    config.afterEditorReady = this.afterEditorReady_.bind(this, areBreakpointsEnabled);
+
+    // Store p5specialFunctions in the unusedConfig array so we don't give warnings
+    // about these functions not being called:
+    config.unusedConfig = this.gameLabP5.p5specialFunctions;
+
+    this.studioApp_.init(config);
+
+    this.debugger_.initializeAfterDomCreated({
+      defaultStepSpeed: 1
+    });
+  }.bind(this);
+
+  this.reduxStore_.dispatch(actions.setInitialLevelProps({
+    assetUrl: this.studioApp_.assetUrl,
+    isEmbedView: !!config.embed,
+    isShareView: !!config.share
+  }));
```

Comment:
I moved the controlsRow stuff down, since it was only being used by generateCodeWorkspaceHtmlFromEjs and generateVisualizationColumnHtmlFromEjs, and it didn't make sense to me to have onMount between them.